common/appscale/common/service_stats/samples.py

```diff
@@ -0,0 +1,54 @@
+""" This module contains standard functions to use in Service Stats. """
+
+
+def categorize_by_app(req_info):
+  return req_info.app
+
+
+def categorize_by_method(req_info):
+  return req_info.method
+
+
+def categorize_by_resource(req_info):
+  return req_info.resource
+
+
+def categorize_by_status(req_info):
+  return req_info.status
+
+
+def summarize_any(req_info):
+  """ Can be also used as matcher. """
+  return True
+
+
+def summarize_client_error(req_info):
+  """ Can be also used as matcher. """
+  return 400 <= req_info.status <= 499
+
+
+def summarize_server_error(req_info):
+  """ Can be also used as matcher. """
+  return 500 <= req_info.status <= 599
+
+
+def summarize_latency(req_info):
+  return req_info.latency
+
+
+def count_any(requests):
```

Comment:
Using "any" is not accurate any more. Previously it was "match any", not is should be "count all". The same for "summarize all".